docs/deploy-from-source-code.md

```diff
@@ -38,81 +83,53 @@
-
-### Run via dotnet CLI
-
-* Open console
-
+## Running
+To run platform by dotnet CLI:
+1. Open console
 ```console
 cd src\VirtoCommerce.Platform.Web
 ```
-
-* Install all required npm packages
-
-   ```console
-    npm ci
-   ```
-
-*- Bundle all js scripts and css styles
-
-```console
-npm run webpack:build
-```
-
-* Run platform by dotnet CLI.
-
+2. Run
 ```console
 dotnet run -c Development --no-launch-profile
 ```
-
 **Note:** you can add `--no-build` flag to speed the start, if you have compiled the solution already.
 
-* Open  `http://localhost:10645` in the browser.
-* On the first request the application will create and initialize database. After that you should see the sign in page. Use the following credentials: `admin/store` to sign in.
-
-**Note:** Don't forget to change them after the first sign in.
-
-## Module from source code getting started
-
-* Run VC Platform  from precompiled binaries or source code as described in the steps above
-* Run command to change the current directory
-
-```console
-cd src\VirtoCommerce.Platform.Web\Modules
-```
 
-* Clone module repository from GitHub into `Modules` folder
+Or run from Visual Studio
+* Open `VirtoCommerce.Platform.sln` 
+* Set VirtoCommerce.Platform.Web as Startup Project
+* Go to Debug > Start Debugging (or Press F5)
 
-```console
-git clone  https://github.com/VirtoCommerce/{module-name.git}  src\VirtoCommerce.Platform.Web\Modules\{module-name}
-```
 
-* Build module code
+## Usage
+* Open `http://localhost:10645` in the browser
+* On the first request the application will create and initialize database. After that you should see the sign in page. Use the following credentials: `admin/store` to sign in
 
-```console
-cd src\VirtoCommerce.Platform.Web\Modules\{module-name}\src\{module-name}.Web
-dotnet build -c Development
-```
+**Note:** Don't forget to change them after the first sign in.
 
-* Install all required npm packages
+## Backend Debugging
+* Install and run platform as described in steps above
+* Open the module solution in Visual Studio and attach the debugger to the `VirtoCommerce.Platform.Web.exe` process
 
-```console
-npm ci
-```
+![image](media/backend-debug.png)
 
-* Bundle all js scripts and css styles
+## Frontend Debugging
+* Frontend supports debugging in Chrome.
+* Open Chrome Developer Tools (Press F12)
+* Open Sources tab
+* Navigate to `{module-name}/./Script/`
+* Debug code
 
-```console
-npm run webpack:build
-```
+![image](media/frontend-debug-chrome.png)
 
-* Restart the Platform to load the new module assemblies into the Platform's application process
+## Testing 
+There is `tests` folder with suites which can be run locally.
 
-## How to debug module
+## IDE Specific Usage
+Some additional tips for developing in specific IDEs.
+### Visual Studio
+Recommend to install [WebPack Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebPackTaskRunner) and run webpack tasks from Visual Studio. 
```

Comment:
Move to line 103

Reply:
I checked that Google and other split IDE and Native. Because Native is more important, IDE just improvements. Then we can add tools for VS Code and ect.